bucket/sozi.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "21.12.06",
+    "description": "Zooming presentation editor and player.",
+    "homepage": "https://sozi.baierouge.fr/",
+    "license": " MPL-2.0",
+    "notes": "Install sozi extras media by running: \"scoop install inkscape-extension-sozi-extras\"",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/sozi-projects/Sozi/releases/download/v21.12.06/sozi-21.12.06-1638805322-windows-x64.zip",
+            "hash": "6642d6077d72d055e91d6ce29d85a6d11f25c8b189dd141636ce2b633a624664",
+            "extract_dir": "sozi-21.12.06-1638805322-windows-x64"
+        }
+    },
+    "shortcuts": [
+        [
+            "sozi.exe",
+            "Sozi"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/sozi-projects/Sozi",
```

Comment:
```suggestion
        "url": "https://github.com/sozi-projects/Sozi/releases",
```

Reply:
Could explain why `url` is used instead of `github` ? is it because I'm using `regex`?

Reply:
Precisely